video/out/gpu/d3d11_helpers.c

```diff
@@ -201,17 +201,22 @@ static IDXGIAdapter1 *get_d3d11_adapter(struct mp_log *log,
             bstr_xappend_asprintf(NULL, listing,
                                   "Adapter %u: vendor: %u, description: %s\n",
                                   adapter_num, desc.VendorId,
-                                  adapter_description ? adapter_description : "<No Description>");
+                                  adapter_description);
         }
 
-        if (requested_adapter_name && adapter_description &&
-            !strcmp(requested_adapter_name, adapter_description))
+        if (adapter_description && requested_adapter_name.len &&
+            bstr_case_startswith(bstr0(adapter_description),
+                                 requested_adapter_name))
```

Comment:
It should probably be documented that this option supports case-insensitive prefixes. I like the idea of being able to use --d3d11-adapter=nvidia/intel, but I don't think any other option in mpv works like this yet, so users might not figure it out.

Reply:
Documented.